packages/next/package.json

```diff
@@ -82,10 +82,12 @@
     "etag": "1.8.1",
     "find-cache-dir": "3.3.1",
     "jest-worker": "24.9.0",
+    "klona": "^2.0.4",
     "loader-utils": "2.0.0",
     "native-url": "0.3.4",
     "node-fetch": "2.6.1",
     "node-html-parser": "1.4.9",
+    "node-libs-browser": "^2.2.1",
```

Comment:
Is this still needed with subasset bundling? I assume yes, but only for webpack 4?

Reply:
Right, again sort of under tree optimization. So important to take small steps with this stuff.